### PR TITLE
Update capistrano to department standards

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -1,12 +1,11 @@
 set :application, 'purl-fetcher'
 set :repo_url, 'https://github.com/sul-dlss/purl-fetcher.git'
-set :user, 'lyberadmin'
 
 # Default branch is :master
 ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
 
-set :home_directory, "/opt/app/#{fetch(:user)}"
-set :deploy_to, "#{fetch(:home_directory)}/#{fetch(:application)}"
+set :home_directory, "/opt/app/lyberadmin"
```
Comment (ndushay) [inline config/deploy.rb]: Is home_directory used anywhere else?

Comment (jcoyne) [inline config/deploy.rb]: Not that I see

Comment (ndushay) [inline config/deploy.rb]: so remove it?

**Comment by jcoyne** [comment]
@ndushay removed

**Pull request merged**